app/models/article/part.rb

```diff
@@ -8,4 +8,13 @@ class Page
 
     default_scope ->{ where(route: "article/page") }
   end
+
+  class PageNavi
+    include Cms::Model::Part
+    include Cms::Addon::Release
+    include Cms::Addon::GroupPermission
+    include History::Addon::Backup
+
+    default_scope ->{ where(route: "article/page_navi") }
```

Comment:
__[rails_best_practices]__
* default_scope is evil [:ref](http://rails-bestpractices.com/posts/2013/06/15/default_scope-is-evil/)

<!-- Comment by [SideCI](https://sideci.com) -->